test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/CaseTraderViewSpec.scala

```diff
@@ -140,6 +140,28 @@ class CaseTraderViewSpec extends ViewSpec {
       doc shouldNot containElementWithID("agent-letter-file-letter-of-auth-id")
     }
 
+    "show the suppress case section for NEW cases" in {
+      // Given
+      val case1 = Cases.btiCaseExample.copy(status = CaseStatus.NEW)
+
+      // When
+      val doc = view(case_trader(case1, None))
+
+      // Then
+      doc should containText("Releasing or suppressing a case")
```

Comment:
Can we assert on IDs rather than content please :)

Reply:
Will do. 

Again I based this on the existing tests in this area

Reply:
Done